pages/common/minifab.md

```diff
@@ -0,0 +1,36 @@
+# minifab
+
+> Utility tool that automates the setup and deployment of Hyperledger Fabric networks.
+> More information: <https://github.com/hyperledger-labs/minifabric>.
+
+- Bring up default Hyperledger Fabric network:
+
+`minifab up -i {{minifab_version}}`
+
+- Bring down Hyperldger Fabric network:
+
+`minifab down`
+
+- Install chaincode onto channel:
+
+`minifab install -n {{chaincode_name}}`
+
+- Install chaincode version onto channel:
+
+`minifab install -n {{chaincode_name}} -v {{chaincode_version}}`
+
+- Initialize chaincode after installation/upgrade:
+
+`minifab approve,commit,initialize,discover`
```

Comment:
Missing token syntax? I assume that only a single word here is used at once?
```suggestion
`minifab {{approve|commit|initialize|discover}}`
```

Reply:
Actually this is a legit command that does all of the commands specified sequentially

Reply:
Oh, I see! You have to actually type them out as-is. Let's leave this bit as-is then.